specification/v1.1.0-rc/OSDM-online-api-v1.1.0-rc.yml

```diff
@@ -3095,6 +3118,21 @@ components:
           type: string
         carrierConstraintText:
           type: string
+        combinationTags:
+          description: >-
+            Tags to indicate that some products from a provider can be sold only when in conjunction with product(s) from another provider using the same tag.
+            At least one, not all, combinationTags must be in common to allow combination. No combinationTags indicate that there are no combination constraints on the product.        
+          type: array
+          items:
+            type: object
+            properties:
+              combinationTag:
+                type: string
+              sellableStandalone:
+                description: >-
```

Comment:
This addition does make sense: with the current specs we cannot differentiate if a product must absolutely be sold in conjunction with another or not